recipes/omniscidb-cuda/recipe/build.sh

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bash
+
+set -ex
+
+# Make sure we are in the right place
+cd $SRC_DIR
+
+export LDFLAGS="-L$PREFIX/lib -Wl,-rpath,$PREFIX/lib"
+
+# libcuda.so is a machine specific library. For building, we use
+# libcuda.so from stubs. For runtime, users must specify the location
+# of libcuda.so.1 in the environment variable LD_LIBRARY_PATH.
+export LDFLAGS="$LDFLAGS -L$PREFIX/lib/stubs -Wl,-rpath-link,$PREFIX/lib/stubs"
```

Comment:
I don't think this should still be needed, but please let me know if it still doesn't work without this.

Reply:
Removed.